osf/models/user.py

```diff
@@ -1393,3 +1396,10 @@ class Meta:
         permissions = (
             ('view_user', 'Can view user details'),
         )
+
+@receiver(post_save, sender=OSFUser)
+def create_bookmark_collection(sender, instance, created, **kwargs):
+    if created:
+        new_bookmark_collection(instance)
```

Comment:
For some reason, creating a bookmark collection when a user is created leads to incorrect contributor permissions for just about everything. It could be that writes aren't happening on the contributor table, or the MRM is caching outdated values, or something else entirely.
```
#TODO: Investigate further tomorrow
```